internal/engine/container_build_and_deployer.go

```diff
@@ -76,13 +76,13 @@ func (cbd *ContainerBuildAndDeployer) BuildAndDeploy(ctx context.Context, servic
 	if err != nil {
 		return BuildResult{}, err
 	}
-	logger.Get(ctx).Infof("Updating container...")
+	logger.Get(ctx).Infof("  → Updating container…")
```

Comment:
I know you said this was left purposefully half-assed. Just a comment that when we make this full-assed we should probably have some programmatic way of indenting/formatting these logs instead of doing it manually.